src/actions/cloud-compute/run-test-run.ts

```diff
@@ -195,6 +202,12 @@ Tunnel will be live for up to ${DEBUG_MODE_KEEP_TUNNEL_OPEN_DURATION.toHuman()}.
     }
   };
 
+  const onTunnelStillLocked = () => {
+    logger.info(
+      "The test run has completed but additional tasks on the Meticulous platform are using this deployment, please keep this job running..."
+    );
+  };
+
   const onTestRunCreated = (testRun: TestRun) => {
     logger.info(`Test run created: ${testRun.url}`);
   };
```

Comment:
So executeRemoteTestRun will hold off from returning until the lock is released?

Reply:
That's right, and periodically call `onTunnelStillLocked` above so that we log something.